app/models/goal.py

```diff
@@ -1,6 +1,19 @@
 from flask import current_app
 from app import db
+from sqlalchemy import Table, Column, Integer, ForeignKey
+from sqlalchemy.orm import relationship
```

Comment:
Hmm... I don't think you need to import sqlalchemy again for your models. `Table`, `Column`, `Integer` etc. should already be coming from the sqlalchemy instance you initialized in `__init__.py` as `db`.